src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs

```diff
@@ -203,6 +203,13 @@ public string ToMinimalDisplayString(SemanticModel semanticModel, int position,
             }
 
             #endregion
+
+            NullableAnnotation ITypeSymbol.NullableAnnotation => this.Nullability;
```

Comment:
> Did we want to put this change into a branch so we can also do the IDE cleanup at the same time? Otherwise we're making wrappers around wrappers.

Merging this change into a feature branch is an option. However, this change is quite big and is likely to encounter many merge conflicts and build/tests breaks due to changes in the relationship between types and interfaces they implement if we keep it on the side for a long time. I experienced this while working on this change. Merging with mater was causing all kinds of breaks.